vowpalwabbit/learner.h

```diff
@@ -472,6 +473,30 @@ struct learner
     VW_WARNING_STATE_POP
   }
 
+  // never called, convienience method in case reduction has a seperate
```

Comment:
never called?

Reply:
good point, the comment might be nonsensical. It's a comment on how this method is never called by the VW machinery, some learner fns get called recursively, others just the top one, this one never gets called by it. It does get called by cb_to_cb_adf so yea, that's a bit weird.